streaming/buildtest.sh

```diff
@@ -60,6 +60,8 @@ function test_streaming_java()
     pushd "$script_dir" || exit
 
     bazel build libstreaming_java.so
+    pushd $script_dir/java
+    bash test.sh
```

Comment:
there are so many verbose painc log on std out. Shall we suppress output like ray repo do that?